src/collection-browser.ts

```diff
@@ -245,70 +249,95 @@ export class CollectionBrowser
+        .placeholderType=${this.emptyPlaceholder}
+        .mobileView=${this.mobileView}
+      ></empty-placeholder>
+    `;
+  }
+
+  private get collectionBrowserTemplate() {
+    return html`<div
+        id="left-column"
+        class="column${this.isResizeToMobile ? ' preload' : ''}"
+      >
+        <div id="mobile-header-container">
+          ${this.mobileView ? this.mobileFacetsTemplate : nothing}
+          <div id="results-total">
+            <span id="big-results-count">
+              ${this.totalResults !== undefined
+                ? this.totalResults.toLocaleString()
+                : '-'}
+            </span>
+            <span id="big-results-label">Results</span>
           </div>
         </div>
-        <div id="right-column" class="column">
-          ${this.searchResultsLoading ? this.loadingTemplate : nothing}
-          <sort-filter-bar
-            .selectedSort=${this.selectedSort}
-            .sortDirection=${this.sortDirection}
-            .displayMode=${this.displayMode}
-            .selectedTitleFilter=${this.selectedTitleFilter}
-            .selectedCreatorFilter=${this.selectedCreatorFilter}
-            .resizeObserver=${this.resizeObserver}
-            @sortChanged=${this.userChangedSort}
-            @displayModeChanged=${this.displayModeChanged}
-            @titleLetterChanged=${this.titleLetterSelected}
-            @creatorLetterChanged=${this.creatorLetterSelected}
-          ></sort-filter-bar>
-
-          ${this.displayMode === `list-compact`
-            ? this.listHeaderTemplate
-            : nothing}
-          ${!this.searchResultsLoading && this.totalResults === 0
-            ? html`
-                <h2>
-                  Your search did not match any items in the Archive. Try
-                  different keywords or a more general search.
-                </h2>
-              `
-            : nothing}
-
-          <infinite-scroller
-            class="${ifDefined(this.displayMode)}"
-            .cellProvider=${this}
-            .placeholderCellTemplate=${this.placeholderCellTemplate}
-            @scrollThresholdReached=${this.scrollThresholdReached}
-            @visibleCellsChanged=${this.visibleCellsChanged}
-          >
-          </infinite-scroller>
+        <div
+          id="facets-container"
+          class=${!this.mobileView || this.mobileFacetsVisible
+            ? 'expanded'
+            : ''}
+        >
+          ${this.facetsTemplate}
         </div>
       </div>
-    `;
+      <div id="right-column" class="column">
+        ${this.searchResultsLoading ? this.loadingTemplate : nothing}
+        ${this.sortFilterBarTemplate}
+        ${this.displayMode === `list-compact`
+          ? this.listHeaderTemplate
+          : nothing}
+        ${this.infiniteScrollerTemplate}
+      </div>`;
+  }
+
+  private get infiniteScrollerTemplate() {
+    return html`<infinite-scroller
+      class="${ifDefined(this.displayMode)}"
+      .cellProvider=${this}
```

Comment:
what's the value of `${this}` ?

Reply:
oh, legacy one, I think current context. :confused: